### PR TITLE
feat(output): HTTP Gzip compression

### PR DESCRIPTION
**Pull request by samcm**

### CODE REVIEW

File: pkg/output/http/config.go [hunk 1]
```diff
@@ -12,6 +12,7 @@ type Config struct {
 	BatchTimeout       time.Duration     `yaml:"batch_timeout" default:"5s"`
 	ExportTimeout      time.Duration     `yaml:"export_timeout" default:"30s"`
 	MaxExportBatchSize int               `yaml:"max_export_batch_size" default:"512"`
+	GzipCompression    bool              `yaml:"gzip_compression" default:"false"`
```
Comment (Savid) [inline pkg/output/http/config.go:15]: non blocking: should this be enum with the strategy of compression? (none, gzip, ...)

```
Compression    CompressionStrategy              `yaml:"compression" default:"none"`
```

**Review by Savid — APPROVED** [review]

File: pkg/output/http/config.go (hunk 1 above)
Comment (samcm) [reply pkg/output/http/config.go:15]: Yeah SGTM

Comment (samcm) [reply pkg/output/http/config.go]: Updated

Comment (Savid) [reply pkg/output/http/config.go]: :sendit:

**Pull request merged**